src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_TupleCreationExpression.cs

```diff
@@ -46,25 +51,25 @@ private BoundNode RewriteTupleCreationExpression(BoundTupleExpression node, Immu
                 var smallestCtor = (MethodSymbol)TupleTypeSymbol.GetWellKnownMemberInType(smallestType.OriginalDefinition,
                                                                                             TupleTypeSymbol.GetTupleCtor(smallestType.Arity),
                                                                                             _diagnostics,
-                                                                                            node.Syntax);
+                                                                                            syntax);
                 if ((object)smallestCtor == null)
                 {
-                    return node;
+                    return null;
                 }
 
                 MethodSymbol smallestConstructor = smallestCtor.AsMember(smallestType);
-                BoundObjectCreationExpression currentCreation = new BoundObjectCreationExpression(node.Syntax, smallestConstructor, smallestCtorArguments);
+                BoundObjectCreationExpression currentCreation = new BoundObjectCreationExpression(syntax, smallestConstructor, smallestCtorArguments);
 
                 if (underlyingTupleTypeChain.Count > 0)
                 {
                     NamedTypeSymbol tuple8Type = underlyingTupleTypeChain.Peek();
                     var tuple8Ctor = (MethodSymbol)TupleTypeSymbol.GetWellKnownMemberInType(tuple8Type.OriginalDefinition,
                                                                                             TupleTypeSymbol.GetTupleCtor(TupleTypeSymbol.RestPosition),
                                                                                             _diagnostics,
-                                                                                            node.Syntax);
+                                                                                            syntax);
                     if ((object)tuple8Ctor == null)
                     {
-                        return node;
+                        return null;
```

Comment:
node would have a wrong type and that could be confusing to the caller if this is done as a part of conversion lowering - it is not clear whether the helper succeeded or not. Now null indicates a failure.

If caller expects and can tolerate the failure by keeping original node it can easily do ``` ?? node ```